internal/api/design/package_.go

```diff
@@ -265,47 +265,33 @@ var _ = Service("package", func() {
 	})
 })
 
-var EnumPackageStatus = func() {
+var EnumSIPStatus = func() {
 	Enum(enums.SIPStatusInterfaces()...)
 }
 
-var Package_ = Type("Package", func() {
-	Description("Package describes a package to be stored.")
-	Attribute("name", String, "Name of the package")
-	TypedAttributeUUID("location_id", "Identifier of storage location")
-	Attribute("status", String, "Status of the package", func() {
-		EnumPackageStatus()
-		Default(enums.SIPStatusNew.String())
-	})
-	AttributeUUID("workflow_id", "Identifier of processing workflow")
-	AttributeUUID("run_id", "Identifier of latest processing workflow run")
-	AttributeUUID("aip_id", "Identifier of AIP")
-	Attribute("created_at", String, "Creation datetime", func() {
-		Format(FormatDateTime)
-	})
-	Attribute("started_at", String, "Start datetime", func() {
-		Format(FormatDateTime)
-	})
-	Attribute("completed_at", String, "Completion datetime", func() {
-		Format(FormatDateTime)
-	})
-	Required("id", "status", "created_at")
-})
-
-var StoredPackage = ResultType("application/vnd.enduro.stored-package", func() {
-	Description("StoredPackage describes a package retrieved by the service.")
-	Reference(Package_)
+var SIP = ResultType("application/vnd.enduro.ingest.sip", func() {
+	Description("SIP describes an ingest SIP type.")
+	TypeName("SIP")
 	Attributes(func() {
-		Attribute("id", UInt, "Identifier of package")
-		Attribute("name")
-		Attribute("location_id")
-		Attribute("status")
-		Attribute("workflow_id")
-		Attribute("run_id")
-		Attribute("aip_id")
-		Attribute("created_at")
-		Attribute("started_at")
-		Attribute("completed_at")
+		Attribute("id", UInt, "Identifier of SIP")
+		Attribute("name", String, "Name of the SIP")
```

Comment:
Nice job adding descriptions to all these attributes. :clap:

Reply:
I just moved them from the type to the result type ;)